src/grantRound.ts

```diff
@@ -0,0 +1,31 @@
+import { BigInt, dataSource } from "@graphprotocol/graph-ts"
+import { AddMatchingFunds } from "../generated/templates/GrantRound/GrantRound"
+import { GrantRoundDonation } from "../generated/schema"
+
+export function handleAddMatchingFunds(event: AddMatchingFunds): void {
+    // extract the grantRounds context
+    const context = dataSource.context()
+
+    // extract the GrantRound address from the context used to create the GrantRound template
+    const address = context.getBytes("address")
```

Comment:
This is just me being unfamiliar with subgraphs, but what is going on here exactly / how is this working? Not sure what `dataSource` and `context` are here

Reply:
Though I probably could just read https://thegraph.academy/developers/defining-a-subgraph/ and figure it out instead of being lazy and asking you haha 😅

Reply:
I thinks in step 9 of that documentation! Essentially, when we create the contract listeners for the GrantRound otf, we do so with a context object, which allows us to pass over arbitrary data to the listeners, like the init args/contract address etc.